test/fixtures/users.yml

```diff
@@ -1,9 +1,31 @@
-# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+admin:
```

Comment:
since @krisberry is a girl)) it's nice to have "factory_girl" instead of fixtures )
process well described here:
https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md
and in google)